gobblin-metrics-libs/gobblin-metrics/src/main/java/org/apache/gobblin/metrics/ServiceMetricNames.java

```diff
@@ -43,6 +43,12 @@ public class ServiceMetricNames {
   public static final String FLOW_TRIGGER_HANDLER_JOB_DOES_NOT_EXIST_COUNT = GOBBLIN_SERVICE_PREFIX + "." + FLOW_TRIGGER_HANDLER_PREFIX + ".jobDoesNotExistInScheduler";
   public static final String FLOW_TRIGGER_HANDLER_FAILED_TO_SET_REMINDER_COUNT = GOBBLIN_SERVICE_PREFIX + "." + FLOW_TRIGGER_HANDLER_PREFIX + ".failedToSetReminderCount";
 
+  // DagManager Related Metrics
+  public static final String DAG_MANAGER_HANDLING_PREFIX = GOBBLIN_SERVICE_PREFIX + ".dagManagerHandling";
```

Comment:
overall, this is minor... not wanting to press too hard here.

just observing that `DagManagerHandling`, not being an actual class name, seems a synonym for `DagMgrWorking`, `DagMgrInAction`, `DagMgrDoingStuff`, which is to say it's vague.  even more, DM "doing stuff" is implied simply by the prefix `DagMgr`.

so if there is something you want to differentiate here, that's fine... but do make sure what that is is clearly stated, so a future maintainer knows whether the metric they wish to add belongs under `DagMgr` or your new prefix.

Reply:
Yes, I see what you're saying. I can't find a better way to differntiate this sub-class of `dagManager` code issues so I'll leave it as `dagManager` to avoid any confusion.